src/js/common/components/RequiredLoginView.jsx

```diff
@@ -9,39 +9,30 @@ import LoadingIndicator from '../../common/components/LoadingIndicator';
 class RequiredLoginView extends React.Component {
   constructor(props) {
     super(props);
-    this.setUserLevel = this.setUserLevel.bind(this);
     this.state = {
       loading: true
     };
   }
 
   componentDidMount() {
-    if (sessionStorage.userToken) {
-      this.setUserLevel();
-    }
-
     setTimeout(() => {
       this.setState({ loading: false });
     }, 2000);
   }
 
-  setUserLevel() {
+  isServiceAvailable() {
     const requiredApp = this.props.serviceRequired;
     const userServices = this.props.userProfile.services;
-    if (userServices) {
-      this.setState(
-        {
-          isServiceAvailableForUse: userServices.includes(requiredApp),
-        }
-      );
-    }
+    // TODO: replace indexOf() once NodeJS versions in all environments support includes()
```

Comment:
Yes, just upgraded locally. I'll try this after everything else is approved, and make sure everything passes both locally and on Jenkins. I can't bear to do it now and possibly see new failures.  😩